BepInEx/Configuration/ConfigFile.cs

```diff
@@ -118,7 +118,7 @@ public void Reload()
 						continue;
 					}
 
-					string[] split = line.Split('='); //actual config line
+					string[] split = line.Split('=',2); //actual config line
```

Comment:
.NET Framework 3.5 does not have an overload `String.Split(char, int)`, so this will not compile. If you go this way, you'd have to pick [an overload that's available in FW 3.5](https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netframework-3.5) as that's the target BepInEx uses.